tap_github/schemas/shared/commits.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Add shared folder import in MANIFEST.in file like below:
`include tap_github/schemas/shared/*.json`

Reply:
Added in manifest